terraform/modules/dbx_uc_create_schema/main.tf

```diff
@@ -0,0 +1,12 @@
+locals {
+  name_postfix = var.env == "prod" ? "" : "-${var.env}"
+}
+
+resource "databricks_schema" "this" {
```

Comment:
Burde vi opprette noen grants til schemaet? Slik at teamet kan bruke schemaet altså, se [denne](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/grants)

Reply:
Tenker da på at schemaet/katalogene bare er synlige for andre, ikke at man kan se dataene nødvendigvis

Reply:
Siden teamet skal være eier av katalogen, så burde permission propageres ned fra catalog nivå til schema nivå. Men da må vi først få på plass at eier av katalogen er teamet.

Reply:
Litt usikker på hvordan vi kan gjøre alle tabeller synelige for andre, samtidig som at de ikke har tilgang til data. Tror vi da beveger oss mot en datakatalog implementasjon. kanskje..